src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -307,9 +314,14 @@ string MakeSureLibraryIsInPlace (string destinationBase, string url, string vers
 					Log.LogMessage (MessageImportance.Low, e.ToString ());
 				}
 			}
-			else
-				LogDebugMessage ("    reusing existing archive: {0}", file);
-			string contentDir = Path.Combine (destinationDir, "content");
+			else {
+				if (string.IsNullOrEmpty (extraPath))
+					LogDebugMessage ("    reusing existing archive: {0}", file);
+				else
+					LogDebugMessage ("    found {0} in {1}", embeddedArchive, Path.Combine (AndroidSdkDirectory, extraPath));
```

Comment:
the message you end up getting is

```found m2repository/com/android/support/support-v4/23.3.0/support-v4-23.3.0.aar in /Users/dean/android-toolchain/sdk/extras/android```

which I think is fairly descriptive. 
So we end up with stuff like this in the Log

```
Making sure we have https://dl-ssl.google.com/android/repository/android_m2repository_r29.zip downloaded and extracted m2repository/com/android/support/support-v4/23.3.0/support-v4-23.3.0.aar from it...

found m2repository/com/android/support/support-v4/23.3.0/support-v4-23.3.0.aar in /Users/dean/android-toolchain/sdk/extras/android

Extracting /Users/dean/android-toolchain/sdk/extras/android/m2repository/com/android/support/support-v4/23.3.0/support-v4-23.3.0.aar to /Users/dean/.local/share/Xamarin/Xamarin.Android.Support.v4/23.3.0.0/embedded

Extracted 2 of 23 files
```